src/components/CreateToDo/CreateToDo.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import "./CreateToDo";
+import "./CreateToDo.scss";
+import { BiPlusCircle } from "react-icons/bi";
+
+const CreateToDo = ({
+  onAdd,
+  stopAdd,
+}: {
+  onAdd: () => void;
+  stopAdd: () => boolean;
```

Comment:
Good work! But I am not sure if we need to pass this function as a props for this component. Can we try to do it in a simpler way?

Reply:
Already done.